<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>30</y>
      <width>261</width>
      <height>121</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Vertical</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:48pt; font-style:italic;&quot;&gt;MLTVA&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="labellogo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>427</width>
      <height>163</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../data/Logo.jpg</pixmap>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>220</y>
      <width>211</width>
      <height>59</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
    border: none;
}
</string>
    </property>
    <property name="title">
     <string>Përiode</string>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QComboBox" name="moisComboBox">
         <item>
          <property name="text">
           <string>Janvier</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Février</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Mars</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Avril</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Mai</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Juin</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Juillet</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Août</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Septembre</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Octobre</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Novembre</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Décembre</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="anneeLineEdit"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBoxpreiode">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>320</y>
      <width>191</width>
      <height>211</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
    border: none;
}
</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="depensesButton">
       <property name="text">
        <string>Dépences</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="recettesButton">
       <property name="text">
        <string>Recettes</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_export_pdf">
       <property name="text">
        <string>Export PDF</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pusharegeler">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;founiseur a&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;regler&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="whatsThis">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Fonineur a&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;Regler&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>A regler</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="quitterButton">
       <property name="text">
        <string>Quitter</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>2</width>
      <height>2</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuConfig">
    <property name="title">
     <string>Config</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionContacts"/>
   </widget>
   <widget class="QMenu" name="menuAide">
    <property name="title">
     <string>Aide</string>
    </property>
   </widget>
   <addaction name="menuConfig"/>
   <addaction name="menuAide"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionContacts">
   <property name="text">
    <string>Contacts</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
